CHANGELOG.asciidoc

```diff
@@ -68,6 +68,7 @@ https://github.com/elastic/beats/compare/v6.0.0-alpha2...master[Check the HEAD d
 - Add support for loading Xpack Machine Learning configurations from the modules, and added sample configurations for the Nginx module. {pull}4506[4506]
 - Add udp prospector type. {pull}4452[4452]
 - Enabled Cgo which means libc is dynamically compiled. {pull}4546[4546]
+- Add module config reloading mechanism {pull}4566[4566]
```

Comment:
We should probably mark this as `beta`, since the config reloading in metricbeat is also beta.